### PR TITLE
Wallet Ui

### PR DESCRIPTION
**Pull request by sajalbnl**
made 3 screens of wallet

### CODE REVIEW

File: wallet_app/android/app/src/main/java/com/example/walletapp/MainActivity.kt [hunk 1]
```diff
@@ -1,20 +1,174 @@
 package com.example.walletapp
```
Comment (trbutler4) [inline wallet_app/android/app/src/main/java/com/example/walletapp/MainActivity.kt:1]: can you refactor to com.snphone.walletapp?

**Review by trbutler4 — CHANGES_REQUESTED** [review]

**Comment by sajalbnl** [comment]
updated dependencies

File: wallet_app/android/app/src/main/java/com/example/walletapp/MainActivity.kt (hunk 1 above)
Comment (trbutler4) [reply wallet_app/android/app/src/main/java/com/example/walletapp/MainActivity.kt:1]: will do this later

**Comment by trbutler4** [comment]
nice work!

**Pull request merged**